src/renderers/shaders/ShaderChunk/gradientmap_pars_fragment.glsl.js

```diff
@@ -18,7 +18,8 @@ vec3 getGradientIrradiance( vec3 normal, vec3 lightDirection ) {
 
 	#else
 
-		return ( coord.x < 0.7 ) ? vec3( 0.7 ) : vec3( 1.0 );
+		vec2 fw = fwidth( coord );
+		return mix( vec3( 0.7 ), vec3( 1. ), smoothstep( 0.7 - fw.x, 0.7 + fw.x, coord.x));
```

Comment:
Have you considered halving the fw when using it as a step in the smoothstep function? As it is the step will happen over two pixels which will result in a less crisp edge than it could be:

```glsl
return mix( vec3( 0.7 ), vec3( 1. ), smoothstep( 0.7 - fw.x * 0.5, 0.7 + fw.x * 0.5, coord.x ) );
```

Also nit: can we use `1.0` instead of `1.` for consistency and make sure we have consistent spaces inside the parens.

Reply:
Indeed! Updated the PR with the changes.